### DISCUSSION TITLE
Image captions in Markdown don't display correctly on GitHub

### DISCUSSION THREAD

**Discussion by worldofgeese** [community/community#128611] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

For example, 

```markdown
<figure>
    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/14/Un_hobbit.jpg/640px-Un_hobbit.jpg" alt="An illustration of a hobbit with hairy feet" />
    <figcaption>This is the image caption</figcaption>
</figure>
```

produces, in Markdown files and comments like this one,

<figure>
    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/14/Un_hobbit.jpg/640px-Un_hobbit.jpg" alt="An illustration of a hobbit with hairy feet" />
    <figcaption>This is the image caption</figcaption>
</figure>

**Comment by TheGiraffe3** [comment]
The alternative text is what displays if you hover, I think. The `figcaption` therefore produces the caption.

**Reply by worldofgeese** [reply]
That is correct, however you can see that the caption is produced unformatted. It's not clear it belongs to the image and any following text after the captioned image does not respect line breaks.

**Reply by TheGiraffe3** [reply]
Can you use the formatting symbols? (_ _, ** **)
